client/src/annoMatrix/loader.ts

```diff
@@ -97,24 +112,28 @@ export default class AnnoMatrixLoader extends AnnoMatrix {
     return newAnnoMatrix;
   }
 
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-  dropObsColumn(col: any) {
+  dropObsColumn(col: LabelType): AnnoMatrix {
     /*
 		drop column from field
 		*/
-    const colSchema = _getColumnSchema(this.schema, "obs", col);
-    _writableCheck(colSchema); // throws on error
+    const colSchema = _getColumnSchema(
+      this.schema,
+      Field.obs,
+      col
+    ) as AnnotationColumnSchema;
+    _writableObsCheck(colSchema); // throws on error
 
     const newAnnoMatrix = this._clone();
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-    newAnnoMatrix._cache.obs = (this as any)._cache.obs.dropCol(col);
+    newAnnoMatrix._cache.obs = this._cache.obs.dropCol(col);
     newAnnoMatrix.schema = removeObsAnnoColumn(this.schema, col);
     return newAnnoMatrix;
   }
 
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'colSchema' implicitly has an 'any' type... Remove this comment to see the full error message
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-  addObsColumn(colSchema, Ctor, value) {
+  addObsColumn<T extends DataframeValueArray>(
+    colSchema: AnnotationColumnSchema,
+    Ctor: new (n: number) => T,
```

Comment:
Possible to use `TypedArrayConstructor`?  From `client/src/common/types/arraytypes.ts:21`!

Reply:
you can, but this is also fine - it is the classic TS style...